example/android/app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- Theme applied to the Android Window while the process is starting -->
-    <style name="LaunchTheme" parent="@android:style/Theme.Black.NoTitleBar">
+    <style name="LaunchTheme" parent="Theme.AppCompat">
         <!-- Show a splash screen on the activity. Automatically removed when

```

Comment:
This change was required to fix showing BiometricPrompt UI in lower Android versions.